pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/pulsar/client/kafka/compat/PulsarClientKafkaConfig.java

```diff
@@ -29,6 +30,8 @@
 
     /// Config variables
     public static final String AUTHENTICATION_CLASS = "pulsar.authentication.class";
+    public static final String AUTHENTICATION_PARAMS_MAP = "pulsar.authentication.params.map";
+    public static final String AUTHENTICATION_PARAMS_STRING = "pulsar.authentication.params.string";
```

Comment:
@ssunorz Can you also add these new keys to the docs page at https://github.com/apache/incubator-pulsar/blob/master/site2/docs/adaptors-kafka.md ?

Reply:
@merlimat Thanks for the review. I updated the document according to your suggestions.